src/containers/RegisterRouting/components/WeightForm.jsx

```diff
@@ -22,10 +22,9 @@ function WeightForm(props) {
                     <input type="number" ref={inputWeightRef} />
                     <span>kg</span>
                 </label>
-                <input type="button" onClick={saveWeight} value="Save" />
             </div>
 
-            <Link to="/register/sex" onClick={onClickHandler}>Continue</Link>
+            <Link to="/register/gender" onClick={onClickHandler} className="btn">Continue</Link>
```

Comment:
```suggestion
            <Link
               to="/register/gender"
               onClick={onClickHandler}
               className="btn"
              >
                  Continue
             </Link>
```

For long list of properties i'd like to format them this way for better readability

Reply:
applied